chart/templates/storageclass.yaml

```diff
@@ -2,8 +2,6 @@ kind: StorageClass
 apiVersion: storage.k8s.io/v1
 metadata:
   name: {{ .Release.Name }}-single-replica
-  labels:
```

Comment:
Why are we removing this label?

Reply:
We don't need to pick out the default storageclass by label, or grab the release name from it (we're using the rest api Pod's label for that). Also the storageclass plays no role in the upgrade, so we need not verify it's version. We might not need this label any more.